monitor/commands/reload.go

```diff
@@ -0,0 +1,75 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/containerd/console"
+	controllererrors "github.com/docker/buildx/controller/errdefs"
+	controllerapi "github.com/docker/buildx/controller/pb"
+	"github.com/docker/buildx/monitor/types"
+	"github.com/pkg/errors"
+)
+
+type ReloadCmd struct {
+	m types.Monitor
+
+	stdout       io.WriteCloser
+	stderr       console.File
+	progressMode string
+
+	options      *controllerapi.BuildOptions
+	invokeConfig controllerapi.InvokeConfig
+}
+
+func NewReloadCmd(m types.Monitor, stdout io.WriteCloser, stderr console.File, progressMode string, options *controllerapi.BuildOptions, invokeConfig controllerapi.InvokeConfig) types.Command {
```

Comment:
Could we use `ioset.In` to bundle these args together?

Reply:
This arguments don't contain stdin and has different stderr type (`stderr console.File`) so maybe we can't use `ioset.In` here.